example/client.properties

```diff
@@ -0,0 +1,6 @@
+scalar.dl.client.cert_holder_id=gopher
+scalar.dl.client.cert_pem=-----BEGIN CERTIFICATE-----\nMIICjTCCAjKgAwIBAgIUOt6KMwZpclftxxQzdJsptjxNjdMwCgYIKoZIzj0EAwIw\nbzELMAkGA1UEBhMCSlAxDjAMBgNVBAgTBVRva3lvMQ4wDAYDVQQHEwVUb2t5bzEf\nMB0GA1UEChMWU2FtcGxlIEludGVybWVkaWF0ZSBDQTEfMB0GA1UEAxMWU2FtcGxl\nIEludGVybWVkaWF0ZSBDQTAeFw0xODA5MTAwODA2MDBaFw0yMTA5MDkwODA2MDBa\nMEUxCzAJBgNVBAYTAkFVMRMwEQYDVQQIEwpTb21lLVN0YXRlMSEwHwYDVQQKExhJ\nbnRlcm5ldCBXaWRnaXRzIFB0eSBMdGQwWTATBgcqhkjOPQIBBggqhkjOPQMBBwNC\nAAT9PMYRxgk1zB7l34GU+7G7CIiEyEqkylMxkz3gv5tGEM8WficyKBwKKOV7LuqE\n5/CmVWtO2c0tM0wPza23CdHuo4HVMIHSMA4GA1UdDwEB/wQEAwIFoDATBgNVHSUE\nDDAKBggrBgEFBQcDAjAMBgNVHRMBAf8EAjAAMB0GA1UdDgQWBBS1Neb+7m9wyM2h\noRs75x86gbzS5TAfBgNVHSMEGDAWgBR+Y+v8yByDNp39G7trYrTfZ0UjJzAxBggr\nBgEFBQcBAQQlMCMwIQYIKwYBBQUHMAGGFWh0dHA6Ly9sb2NhbGhvc3Q6ODg4OTAq\nBgNVHR8EIzAhMB+gHaAbhhlodHRwOi8vbG9jYWxob3N0Ojg4ODgvY3JsMAoGCCqG\nSM49BAMCA0kAMEYCIQDxqVzuhLWxnX6fajucPcjCcvtWTl/4fAAN/n8Py1qmfwIh\nAKtP641f4dGZTV0R6uMYDrZjunwbG+kmt9+vSuE8rjO0\n-----END CERTIFICATE-----\n
+scalar.dl.client.private_key_pem=-----BEGIN EC PRIVATE KEY-----\nMHcCAQEEIIbElY/Vs5nEoVGsHZ8G9icxZcsRlT2DcHIfxFNkZrnHoAoGCCqGSM49\nAwEHoUQDQgAE/TzGEcYJNcwe5d+BlPuxuwiIhMhKpMpTMZM94L+bRhDPFn4nMigc\nCijley7qhOfwplVrTtnNLTNMD82ttwnR7g==\n-----END EC PRIVATE KEY-----\n
```

Comment:
Just to make sure, just a sample private key that cannot be misused? :)

Reply:
It's OK w
Thanks for the reminder.
I think I copied it from the example of another SDK project.